src/components/SharedStyles.jsx

```diff
@@ -0,0 +1,13 @@
+import styled from "@emotion/styled";
+
+const Input = styled.input`
+  border: none;
+  outline: none;
+  flex-grow: 1;
+`;
+
+const Icon = styled.img`
+  padding: ${(props) => props.theme.layout.space200};
+`;
+
+export { Input, Icon };
```

Comment:
I understand your intention in creating a separate `SharedStyles` file, but since these styled components are only used in the `FormInput` component, let's move these definitions into that component file.